saleor/account/migrations/0057_clear_user_addresses.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 3.2.12 on 2022-02-15 14:18
+
+from django.db import migrations
+from django.db.models import Count
+
+USER_ADDRESS_LIMIT = 100
```

Comment:
Shouldn't be `settings.MAX_USER_ADDRESSES = int(os.environ.get("MAX_USER_ADDRESSES", 100))` ?